.github/workflows/static.yml

```diff
@@ -111,7 +111,7 @@ jobs:
         run: |
            if [ "$GITHUB_REF_TYPE" == "tag" ]; then export FRANKENPHP_VERSION=${GITHUB_REF_NAME:1}; else export FRANKENPHP_VERSION=$GITHUB_SHA; fi
            echo "CGO_CFLAGS=-DFRANKENPHP_VERSION=$FRANKENPHP_VERSION $(./buildroot/bin/php-config --includes | sed s#-I/#-I$PWD/buildroot/#g)" >> "$GITHUB_ENV"
-           echo "CGO_LDFLAGS=-framework CoreFoundation -framework SystemConfiguration $(./buildroot/bin/php-config --ldflags) $(./buildroot/bin/php-config --libs)" >> "$GITHUB_ENV"
+           echo "CGO_LDFLAGS=-framework CoreFoundation -framework SystemConfiguration $(./buildroot/bin/php-config --ldflags) -Wl,--start-group $(./buildroot/bin/php-config --libs) -Wl,--end-group" >> "$GITHUB_ENV"
```

Comment:
There is also this line to update (we have to use CircleCI for Apple Silicon builds): https://github.com/dunglas/frankenphp/blob/main/.circleci/config.yml#L42C108-L42C108

Reply:
I'm not familiar with the linker on Apple Silicon. Do you know if it's necessary? New linkers typically already search libs in both directions.

Reply:
I don't know 😅 It's definitely a recent version of clang/LLVM.
However, what's sure is that it's the same version of clang/LLVM on GitHub Actions (even if running on Intel) and CircleCI. So if it's not necessary, it's not necessary to update the docs and the GHA Mac build either.

Reply:
I'll assume it's still using gnu-ld, can't find reliable information online and don't have a mac to test it on.

Reply:
It's using `ldd`, the LLVM linker: https://lld.llvm.org/

Reply:
> I don't know 😅 It's definitely a recent version of clang/LLVM. However, what's sure is that it's the same version of clang/LLVM on GitHub Actions (even if running on Intel) and CircleCI. So if it's not necessary, it's not necessary to update the docs and the GHA Mac build either.

gcc/gnu-ld never changed the default functionality, even on recent versions, that's why they still have `-( -)` or `--start-group --end-group`. gold, lld and vc switched to it by default (or released with it).